examples/java/src/main/java/com/pelias/importer/BostonSchools.java

```diff
@@ -64,7 +64,7 @@ public void process() throws ClientProtocolException, IOException {
 
         // setup connection to Elasticsearch
         TransportClient elasticSearchClient = TransportClient.builder().build().addTransportAddress(
-                new InetSocketTransportAddress(InetAddress.getByName("localhost"), 9300));
+                new InetSocketTransportAddress(InetAddress.getByName("elasticsearch"), 9300));
```

Comment:
Ah i see. The Elasticsearch Java client is special, and actually has much more tight integration with Elasticsearch (since they are both written in Java). https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/transport-client.html